tests/scaletest/lib/utils.go

```diff
@@ -53,6 +53,10 @@ func (e *errorString) Error() string {
 	return e.s
 }
 
+func GetUriEncoded(uri string) string {
+	return strings.ReplaceAll(uri, " ", "%20")
```

Comment:
can we look into url.PathEscape or url.QueryEscape to take care of special characters as well. This has been mentioned in the JIRA ticket to encode.

Reply:
updated code to use url.QueryEscape